cmd/minikube/cmd/delete.go

```diff
@@ -51,6 +57,15 @@ associated files.`,
 	Run: runDelete,
 }
 
+func init() {
+	deleteCmd.Flags().Bool(purge, false, "Set this flag to delete the '.minikube' folder from your user directory. This will prompt for confirmation.")
+	deleteCmd.Flags().Bool(noPrompt, false, "Set this flag so that there are no prompts.")
```

Comment:
Do you mind naming this flag to `--interactive` for consistency with `minikube start`?

Reply:
As per the other feedback, `--interactive` has been removed and instead a combination of switches is now required.